src/utils/renderHtmlDocument.ts

```diff
@@ -5,7 +5,111 @@ interface TemplateData {
     css: string;
 }
 
-export const renderHtmlDocument = ({ html, css }: TemplateData) =>
+export const fontFaces = `
+    @font-face {
```

Comment:
I did talk with @guardian/design-system Simon about including something like this in the Design System but not sure whether they'd be happy to provide this or not?

Reply:
That would be cool but I don't think they export that yet. I'd say let's do with this and I'll speak to Simon and see if there's any work being done around this.